rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -474,6 +480,19 @@ maas_excluded_checks: []
 
 
 #
+# Process list for each process check
```

Comment:
These variables should be eliminated if they can't be used in process_checks_list